Make subflows optional with run_deployment (#11611)
PrefectHQ/prefect: docs/concepts/flows.md

resume_flow_run(FLOW_RUN_ID)
```

> !!! note "Subflows cannot be suspended independently of their parent run"

Reviewer: ```suggestion
!!! note "Subflows can't be suspended independently of their parent run"
```

    You can't suspend a subflow run independently of its parent flow run.

>     If you are using a flow to schedule a flow run with `run_deployment`, the

Reviewer 2: ```suggestion
    If you use a flow to schedule a flow run with `run_deployment`, the
```

>     scheduled flow run will be linked to the calling flow by default. This means

Reviewer: ```suggestion
    scheduled flow run will be linked to the calling flow as a subflow run by default. This means
```
